doc/api/fs.md

```diff
@@ -3792,7 +3792,7 @@ this API: [`fs.write(fd, string...)`][].
 
 The `fs.promises` API provides an alternative set of asynchronous file system
 methods that return `Promise` objects rather than using callbacks. The
-API is accessible via `require('fs').promises`.
+API is accessible via `require('fs').promises` and `require('fs/promises')`.
```

Comment:
this constitutes a new core module - and there's not supposed to be any new ones until https://github.com/nodejs/node/pull/21551#issuecomment-451550904 is resolved. that's also why `fs/promises` was reverted.

Reply:
@ljharb that is not why `fs/promises` was reverted. @jasnell can chime in but if I recall there were security concerns